File: 1.Python/Program21.py
# 9. Write a program which accept number from user and return number of
# digits in that number.
# Input : 5187934
# Output : 7
def DigitCont(No):
     iCnt=0
     while No!=0:
       iCnt = iCnt + 1
       No=No//10
     return iCnt
def Main():
   No=0
   iRet=0
   print("Enter the one number:")
   No=int(input())
   iRet=DigitCont(No)
   print("Your Digit count is: ",iRet)
if __name__ == "__main__":
   Main()

File: 0.C_CPP_JAVA Programs/JAVA/Program21.java
/*
1. Write Java program which accept number of rows and number of columns from user and display below pattern.
Input :  iRow = 4  iCol = 4
Output : A B C D
         A B C D
         A B C D
         A B C D
 */
import java.lang.*;
import java.util.*;

class DemoString
{
    void Pattern(int No1,int No2)
    {
        char ch='A';
        for (int i=0;i<No1;i++)
        {
            for(int j=0;j<No2;j++)
            {
                System.out.print("\t" + ch);
                ch++;
            }
            ch='A';
            System.out.print("\n");
        }
    }
}
class Demo
{
    public static void main(String arg[])
    {

        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the Coloumn:");
        int No1=sobj.nextInt();

        System.out.println("Enter second row:");
        int No2=sobj.nextInt();

        DemoString dobj=new DemoString();

         dobj.Pattern(No1,No2);
    }
}